python_modules/dagster-graphql/dagster_graphql/implementation/external.py

```diff
@@ -34,17 +31,6 @@ def get_external_pipeline_or_raise(graphene_info, selector):
     if selector.solid_selection is None:
         return full_pipeline
 
-    for solid_name in selector.solid_selection:
-        if not full_pipeline.has_solid_invocation(solid_name):
-            raise UserFacingGraphQLError(
-                GrapheneInvalidSubsetError(
-                    message='Solid "{solid_name}" does not exist in "{pipeline_name}"'.format(
-                        solid_name=solid_name, pipeline_name=selector.pipeline_name
-                    ),
-                    pipeline=GraphenePipeline(full_pipeline),
-                )
-            )
-
```

Comment:
removing because 1) this is guarded deeper in the stack when resolving the selection which gets populated in `get_subset_external_pipeline` below; 2) with nested selection, we no long can check this in one-pass iteration.